lib/session_encryptor.rb

```diff
@@ -1,18 +1,128 @@
 class SessionEncryptor
+  class SensitiveKeyError < StandardError; end
+  SENSITIVE_KEYS = [
+    'first_name', 'middle_name', 'last_name', 'address1', 'address2', 'city', 'state', 'zipcode',
+    'zip_code', 'dob', 'phone', 'phone_number', 'ssn', 'prev_address1', 'prev_address2',
+    'prev_city', 'prev_state', 'prev_zipcode', 'pii', 'pii_from_doc', 'password', 'personal_key'
+  ]
```

Comment:
Since we do inclusion checks on this

```suggestion
  ].to_set.freeze
```

Could also switch to `%w[]` syntax if we wanted